CarRecognition/Source Files/Application/ApplicationConstants.swift

```diff
@@ -44,4 +44,16 @@ struct Constants {
         /// Number of last results that should be normalized.
         static let normalizationCount = 5
     }
+    
+    /// Constants used in detection process.
+    struct Detection {
```

Comment:
It will make sense if you will create some additional constants concerning `Detection`. If you take a look into the file itself, most of the struct have nested structs (one or more) and may have additional constants. But yes, I will remove it for now, as it is unknown when new constants will appear :)
Still my proposal would be to change it to the ``` struct Labels {
struct Recognition {..} 
struct Detection {..} 
} ``` :)